docs/docs/quick-start/docusaurus.md

```diff
@@ -14,7 +14,7 @@ Follow these steps to generate Docusaurus previews for each pull request.
         npm run build
 
     - name: Run Stoat Action
-      uses: stoat-dev/stoat-action@v1
+      uses: stoat-dev/stoat-action@v0
```

Comment:
This repo was using `v1`? But we don't have a `v1`.

Reply:
It was just in the docs, not actually used.

Reply:
Oh, right, I just realized that.